cmd/manager/main.go

```diff
@@ -161,9 +175,49 @@ func main() {
 			},
 		}
 	}
+
+	// Force serving be disabled by default if no certs are provided
```

Comment:
This comment is actually describing line 184, right?

Reply:
I will improve the comment to bring more clarity. That might be a good point. 🥇